arbeitszeit_web/www/presenters/list_registered_hours_worked_presenter.py

```diff
@@ -0,0 +1,38 @@
+from dataclasses import dataclass
+
+from arbeitszeit.use_cases import list_registered_hours_worked
+from arbeitszeit_web.formatters.datetime_formatter import DatetimeFormatter
+
+
+@dataclass
+class RegisteredHoursWorked:
+    hours: str
+    worker_name: str
+    worker_id: str
+    registered_on: str
+
+
+@dataclass
+class ViewModel:
+    registered_hours_worked: list[RegisteredHoursWorked]
+
+
+@dataclass
+class ListRegisteredHoursWorkedPresenter:
+    datetime_formatter: DatetimeFormatter
+
+    def present(self, response: list_registered_hours_worked.Response) -> ViewModel:
+        registered_hours_worked = [
+            RegisteredHoursWorked(
+                hours=str(round(record.hours, 2)),
+                worker_name=record.worker_name,
+                worker_id=str(record.worker_id),
+                registered_on=self.datetime_formatter.format_datetime(
+                    date=record.registered_on,
+                    zone="Europe/Berlin",
+                    fmt="%d.%m.%Y %H:%M",
```

Comment:
Same here. For another PR. Formatting could be set via a [language code](https://www.fincher.org/Utilities/CountryLanguageList.shtml) environment variable, like: `de`, `de-DE`, `en`, `en-GB`, `en-US`  of shape: `\<language\>` or `\<language\>-\<country\>`. Even better practice (like above) is that users should be able to set their specific language in their account settings by themselves.